### ISSUE TITLE
tvtropes (zimit) cannot be scraped anymore

### ISSUE THREAD

**Issue by zach-sb** [openzim/zim-requests#1007]
When using the TV Tropes tvtropes_en_all_2023-09.zim that I downloaded today, many pages result in what looks like a TV Tropes 403 error.

![image](https://github.com/openzim/zim-requests/assets/30288956/aa7c134d-0f28-4180-b67c-0f78e0c10786)

For example, starting on the home page:
![image](https://github.com/openzim/zim-requests/assets/30288956/aa8fc22f-26b1-473a-a75f-c5c1b33ca8d4)

Click on  "God", then the first link from there is "The Alpha and the Omega", which results in a 403.
Same thing happens if you start with "great power", Then "Powers that Be". Or anything else on that page that I tried.

This happens with a kiwix-server docker setup, and on the library.kiwix.org website

**Comment by benoit74** [comment]
It is publishing in dev for now. Anyway, task has failed again, blocked right away by the website.

**Comment by benoit74** [comment]
Zimit2 and Crawler 1.x won't help, we are still blocked by the website.
Only solution will be to obtain an whitelisting from website owner. Any point of contact?

**Comment by kelson42** [comment]
@benoit74 @Popolechien It seems this ZIM file does not meet our quality criterias. Should we not just remove the ZIM file if we are not able to fix the problem?

**Comment by benoit74** [comment]
ZIM file is already unpublished, isn't it?

**Comment by kelson42** [comment]
@benoit74 It is, see https://farm.openzim.org/pipeline/43fbde27-4454-466b-85c4-862f8137c16b

**Comment by benoit74** [comment]
![Image](https://github.com/user-attachments/assets/17596b7b-82b9-4160-a89c-ec852990ace0)

¯\_(ツ)_/¯

**Comment by kelson42** [comment]
removing recipe and closing issue is what is left to do?

**Comment by Popolechien** [comment]
I've randomly re-tried the recipe and we got it to work. But it freezes on the cookie banner and the site can't be explored.

<img width="1906" height="1033" alt="Image" src="https://github.com/user-attachments/assets/a2289673-3408-47f3-9147-80cd7b9ff83b" />

**Comment by benoit74** [comment]
ZIM is only 92MB so I doubt it captured the whole website at all.

**Comment by Popolechien** [comment]
Yeah it's about half the size from the last working iteration, but that was 3 years ago.
